cli/exchange/business.go

```diff
@@ -86,6 +86,17 @@ func BusinessAddPolicy(org string, credToUse string, policy string, jsonFilePath
 		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("Incorrect deployment policy format in file %s: %v", jsonFilePath, err))
 	}
 
+	cons := len(policyFile.Constraints)
```

Comment:
Is checking the length of the constraints array enough? What if all the elements in the array are the empty string?

Reply:
Sounds over protecting. But I did it.